### DISCUSSION TITLE
Can't download ISO

### DISCUSSION THREAD

**Discussion by sbrooks-chroma** [Security-Onion-Solutions/securityonion#4110] [Unsupported Versions] [answered]
Multiple attempts to download the ISO from https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md have failed today. Tried different machines on different networks, no AV, no firewall. Anyone else seeing this problem?

**Comment by gcsrellis** [comment, marked as answer]
The link is still working for me.

**Reply by sbrooks-chroma** [reply]
Interestingly, it appears to be working for me now, too. Gremlins in the works earlier, perhaps.
